src/main/java/com/google/firebase/auth/Tenant.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.auth;
+
+import com.google.api.client.util.Key;
+import com.google.auto.value.AutoValue;
+
+/**
+ * Contains metadata associated with a Firebase tenant.
+ *
+ * <p>Instances of this class are immutable and thread safe.
```

Comment:
`<p>` typo? or missing `</p>`?

Reply:
Please ignore this. Just realized how it works in javadoc :)

Reply:
This was intentional and adheres to standard Javadoc formatting.

The closing tag is unnecessary for certain tag types (e.g. <p>, <li>): https://stackoverflow.com/questions/33787753/javadoc-closing-tags.